backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_privileges.py

```
@@ -284,3 +283,21 @@ def test_deactivate_privilege_not_found(self):
         # Note lack of self._load_provider_data() here - we're _not_ loading the provider in this case
         resp = self._request_deactivation_with_scopes('openid email aslp/admin')
         self.assertEqual(404, resp['statusCode'])
+
+    def test_privilege_purchase_message_handler_sends_email(self):
+        """
+        If a valid event purchase event is passed into the privilege_purchase_message_handler, it should kick off the
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)